### PR TITLE
Bump parity-common

### PR DESCRIPTION
**Pull request by pgherveou**
Depends on 
- https://github.com/paritytech/parity-common/pull/860
- https://github.com/Snowfork/ethabi-decode/pull/11

### CODE REVIEW

File: bridges/primitives/polkadot-core/src/parachains.rs [hunk 1]
```diff
@@ -71,7 +68,7 @@ impl From<u32> for ParaId {
 #[derive(
 	PartialEq, Eq, Clone, PartialOrd, Ord, Encode, Decode, RuntimeDebug, TypeInfo, Default,
 )]
-#[cfg_attr(feature = "std", derive(Serialize, Deserialize, Hash, MallocSizeOf))]
```
Comment (pgherveou) [inline bridges/primitives/polkadot-core/src/parachains.rs:74]: should we roll this back, not sure what this do 🙈 , but I saw that the crate was not maintained anymore (removed from parity-common)

Comment (ordian) [reply bridges/primitives/polkadot-core/src/parachains.rs:74]: how are you using mallocsizeof if at all? it was removed a long time ago https://github.com/paritytech/substrate/issues/12658

Comment (pgherveou) [reply bridges/primitives/polkadot-core/src/parachains.rs:74]: so this is a left over right?

**Comment by ordian** [comment]
```
$ cargo tree -i -p primitive-types@0.12.1 | less

primitive-types v0.12.1
└── sp-runtime-interface v17.0.0 (https://github.com/paritytech/polkadot-sdk#82912acb)
    └── sp-crypto-ec-utils v0.4.1 (https://github.com/paritytech/polkadot-sdk#82912acb)
        ├── sp-ark-bls12-381 v0.4.2 (https://github.com/paritytech/arkworks-substrate#caa2eed7)
        │   └── bandersnatch_vrfs v0.0.4 (https://github.com/w3f/ring-vrf?rev=0fef826#0fef8266)
        │       └── sp-core v28.0.0 (/Users/andronik/code/polkadot-sdk/substrate/primitives/core)
```
Not sure what is the right sequence of updating these deps anymore
This looks like a circular dependency to me 😬 https://github.com/paritytech/arkworks-substrate/blob/caa2eed74beb885dd07c7db5f916f2281dad818f/Cargo.toml#L32

**Review by ordian — APPROVED** [review]

File: Cargo.toml [hunk 2]
```diff
@@ -737,9 +737,9 @@ enumflags2 = { version = "0.7.7" }
 enumn = { version = "0.1.13" }
 environmental = { version = "1.1.4", default-features = false }
 equivocation-detector = { path = "bridges/relays/equivocation" }
-ethabi = { version = "1.0.0", default-features = false, package = "ethabi-decode" }
-ethbloom = { version = "0.13.0", default-features = false }
-ethereum-types = { version = "0.14.1", default-features = false }
+ethabi = { version = "1.1.0", default-features = false, package = "ethabi-decode" }
```
Comment (ordian) [inline Cargo.toml:740]:
```suggestion
ethabi = { version = "2.0.0", default-features = false, package = "ethabi-decode" }
```

**Comment by pgherveou** [comment]
dup https://github.com/paritytech/polkadot-sdk/pull/5886

**Pull request closed without merging**